src/Conversion/ONNXToKrnl/RNN/RNNBase.cpp

```diff
@@ -19,9 +19,6 @@ using namespace mlir;
 
 namespace onnx_mlir {
 
-// Check a Value's type is none or not.
```

Comment:
excellent, by removing the function, we can be sure it is not used anymore. I am assuming that you removed all definitions and uses.